Sources/CardPayments/Models/CardResult.swift

```diff
@@ -11,5 +11,6 @@ public struct CardResult {
 
     //// :nodoc: This is the deep link url returned from 3DS authentication
```

Comment:
```suggestion
    //// This is the deep link url returned from 3DS authentication
```

We can remove the `nodoc` word from the docstrings. These never actually did anything in this SDK since it isn't set up for Jazzy.